src/App.css

```diff
@@ -1,4 +1,116 @@
 body {
+  background-color: #FFF;
+  font-family: Verdana, Geneva, Tahoma, sans-serif;
+  min-width: 390px;
+}
```

Comment:
Don't set styles for any tag in component's style files. It should be inside in index.css file.